### PR TITLE
REE Measures

### PR DESCRIPTION
**Pull request by francescpuig7**
### Description:

> Add doc for new REE measures module. This contains, REE files, profiles, CCH curves, TM curves...

### Tasks:

- [x] Add text
- [x] Add images
- [x] Translate
- [x] Fix CCH fact again
- [x] Measures validations
- [x] Attachments

### Links:

> Add links for each file modified

* [Mesures](http://builds.gisce.net/powerp-docs/powerp_measures/ca/distri/medidas/)
* [Medidas](http://builds.gisce.net/powerp-docs/powerp_measures/es/distri/medidas/)

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_measures/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_measures/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: docs/distri/medidas.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
+# Mesures REE
+
+## Mòdul de Mesures REE
+
+Aquest mòdul serveix per la generació de fitxers d'intercanvi de mesures amb
```
Comment (jaumef) [inline docs/distri/medidas.md]: No s'ha traduit correctament

File: docs/distri/medidas.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+REE, siguin d'origen Telegestió, Telemesura o Perfilat.
+Inclou generació, processament i validació d'aquests fitxers.
+
+## Menú de Mesures REE
+
+En el menú de Mesures REE que segueix a la imatge, s'hi troben els següents
```
Comment (jaumef) [inline docs/distri/medidas.md]: En ES: `siguie` → `sigue`

File: docs/distri/medidas.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+apartats.
+
+* Perfils: permet visualitzar els perfils horaris generats a filtrar per
```
Comment (jaumef) [inline docs/distri/medidas.md]: No s'ha traduit correctament

File: docs/distri/medidas.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+* Períodes de Mesures: mostra l'estat, i el progrés, dels períodes de mesures
+* Tancaments: visualitza les dates d'entrega dels fitxers de REE (M+1, M+3...)
+* Casos de perfilació: casos CRM que reporten el comportament al perfilar
+* Casos de mesures: casos CRM que reporten el comportament al generar els
+fitxers de mesures
+* Crear períodes de Mesures: assistent que permet crear els períodes segons any
+
+![](_static/medidas/menu_general.png)
+
+## Preparació de les dades
```
Comment (jaumef) [inline docs/distri/medidas.md]: No s'ha traduit correctament

File: docs/distri/medidas.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+
+Abans de generar els fitxers de mesures, cal comprovar que les dades a incloure
+en aquests, estan preparades. Els orígens de dades són els que segueixen:
+
+### Telegestió:
+corbes provinents de contadors telegestionats, corbas reals, que s'hauran
```
Comment (jaumef) [inline docs/distri/medidas.md]: Comença a meitat la frase?, no ho veig clar...

```suggestion

Per les corbes provinents de contadors telegestionats, corbas reals, que s'hauran
```

File: docs/distri/medidas.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+* Crear períodes de Mesures: assistent que permet crear els períodes segons any
+
+![](_static/medidas/menu_general.png)
+
+## Preparació de les dades
+
+Abans de generar els fitxers de mesures, cal comprovar que les dades a incloure
+en aquests, estan preparades. Els orígens de dades són els que segueixen:
+
+### Telegestió:
```
Comment (jaumef) [inline docs/distri/medidas.md]: Títol sense ":"
```suggestion
### Telegestió
```

File: docs/distri/medidas.md [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+el consum real.
+La mateixa factura indica si la corba té el fix fet, mitjançant el camp
+`CCH disponible`.
+Per assegurar-se que no ha quedat cap corba sense ajustar-se, existeix un
+llistat a:
+**Facturació > General > Factures Client > Factures Client CCH no disponible**
+
+![](_static/medidas/factures_cch_no_disponible.png)
+
+### Telemesura:
```
Comment (jaumef) [inline docs/distri/medidas.md]: Títol sense ":"
```suggestion
### Telemesura
```

File: docs/distri/medidas.md [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+
+Les dades de telemesura ens poden venir de diversos orígens, genèricament es
+carreguen les corbes de telemesura mitjançant fitxers llegits directament del
+comptador per tal de tenir la corba real. Aquesta explicació bé recollida en un
+altre apartat d'aquest manual. És important carregar aquestes corbes el més
+aviat millor.
+
+### Perfil:
```
Comment (jaumef) [inline docs/distri/medidas.md]: Títol sense ":"
```suggestion
### Perfil
```

**Comment by jaumef** [comment]
@francescpuig7 Reminder to end docs!
Also, fix conflicts

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_measures/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_measures/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_measures/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: docs/distri/medidas.md (hunk 5 above)
Comment (francescpuig7) [reply docs/distri/medidas.md]: Done via: https://github.com/gisce/powerp-docs/pull/349/commits/41455280852371ec2510d896f7cb891a233aff92

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_measures/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Pull request merged**